static/js/theme-map/ThemeMap.js

```diff
@@ -294,23 +294,23 @@ class ThemeMap extends ANSWERS.Component {
         this.config.pinClusterClickListener();
       })
       .withIconTemplate('default', (pinDetails) => {
-        return getEncodedSvg(this.config.pinClusterImages.getDefaultPin(pinDetails.pinCount));
+        return getEncodedSvg(this.config.pinClusterImages.getDefaultPin(pinDetails.pinCount).svg);
       })
       .withIconTemplate('hovered', (pinDetails) => {
-        return getEncodedSvg(this.config.pinClusterImages.getHoveredPin(pinDetails.pinCount));
+        return getEncodedSvg(this.config.pinClusterImages.getHoveredPin(pinDetails.pinCount).svg);
       })
       .withIconTemplate('selected', (pinDetails) => {
-        return getEncodedSvg(this.config.pinClusterImages.getSelectedPin(pinDetails.pinCount));
+        return getEncodedSvg(this.config.pinClusterImages.getSelectedPin(pinDetails.pinCount).svg);
       })
       .withPropertiesForStatus(status => {
+        const defaultPinDimensions = this.config.pinClusterImages.getDefaultPin();
```

Comment:
nit: `defaultPinDimensions` is a little misleading since it also contains the SVG. Thoughts on just using `defaultPin`?

Reply:
I wanted to make it clear you should only use this variable for the dimensions, since the SVG depends on pinCount (which I wasn't passing in). Anyways it's passed in now